pyomyo.py

```diff
@@ -43,6 +43,8 @@
 import sys
 import threading
 import time
+import numpy as np
+import pandas as pd
```

Comment:
please fix -- this is not maintainable, you need to add another variable

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Not maintainable

Reply:
ok now it looks good -- it's readable now

---
**Sema Reaction:** :ok_hand: This code looks good | **Sema Tags:** Readable

Reply:
test 9/28/2021 12:03

Reply:
fix

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Inelegant, Not reusable

Reply:
Awesome, Elegant

---
**Sema Reaction:** :trophy: This code is awesome | **Sema Tags:** Elegant

Reply:
ooooh wow